k8_cortx_cloud/cortx-cloud-helm-pkg/cortx-configmap/templates/_config.tpl

```diff
@@ -90,7 +90,7 @@ cortx:
       endpoints: {{- toYaml .Values.cortxMotr.confdEndpoints | nindent 6 }}
     clients:
     {{- if .Values.cortxRgw.enabled }}
-    - name: rgw
+    - name: rgw_s3
```

Comment:
Why is the significance of this name? It is not documented anywhere that I can find. The sample file has also not been updated for this.

https://github.com/Seagate/cortx-prvsnr/blob/2fc7486a652db805619525fab5c26bed98d5a22b/conf/solution/config.yaml.sample#L109